packages/calypso-e2e/src/lib/pages/feedback-inbox-page.ts

```diff
@@ -180,7 +180,7 @@ export class FeedbackInboxPage {
 			.getByRole( 'tab', { name: new RegExp( folderName, 'i' ) } )
 			.or(
 				this.page.getByRole( 'radio', {
-					name: new RegExp( `^${ folderName }\\s*\\(\\d+,\\)$`, 'i' ),
+					name: new RegExp( `^${ folderName }\\s*\\(\\d{1,3}(?:,\\d{3})*\\)$`, 'i' ),
```

Comment:
I believe we could just do this:
```suggestion
					name: new RegExp( `^${ folderName }`, 'i' ),
```

Or better yet (since [Playwright does a case-insensitive substring search by default](https://github.com/microsoft/playwright/issues/27483#issuecomment-1750876265)), this:
```suggestion
					name: folderName,
```